wandb/proto/wandb_internal.proto

```diff
@@ -46,7 +46,7 @@ message Record {
     RunPreemptingRecord preempting = 23;
     LinkArtifactRecord link_artifact = 24;
     UseArtifactRecord use_artifact = 25;
-    LaunchWandbConfigParametersRecord wandb_config_parameters = 26;
+
```

Comment:
technically you should reserve this one, but assuming we only going to use it together maybe it is fine to violate this

Reply:
Yeah not very hygenic, but I think it is fine to violate this as long as we don't add a new record type at 26 before the next core release